### PR TITLE
fix(chat/messages): use timestamp instead of whisperTimestamp

### PR DESCRIPTION
**Pull request by osmaczko**
### What does the PR do

- use timestamp instead of whisperTimestamp (see https://github.com/status-im/status-desktop/issues/8962#issuecomment-1384403122)

fixes: #8962

### Affected areas

- messages' datetime

### CODE REVIEW

File: src/app_service/service/message/dto/message.nim [hunk 1]
```diff
@@ -83,7 +83,6 @@ type TransactionParameters* = object
 type MessageDto* = object
   id*: string
   communityId*: string
-  whisperTimestamp*: int64
```
Comment (jrainville) [inline src/app_service/service/message/dto/message.nim:86]: You can leave it in the dto if you want

**Review by jrainville — APPROVED** [review]

File: src/app_service/service/message/dto/message.nim (hunk 1 above)
Comment (osmaczko) [reply src/app_service/service/message/dto/message.nim:86]: Initially did it that way, with the comment `# don't use`, then decided to remove it altogether. Will bring it back if ever needed.

**Review by caybro — APPROVED** [review]
LGTM

**Review by saledjenic — APPROVED** [review]

**Comment by osmaczko** [comment]
@saledjenic @jrainville @caybro re-requesting review cause I added new crucial commit:

[fix(chat/messages): ensure there is only one label for given date](https://github.com/status-im/status-desktop/pull/9146/commits/41213486eb39acc25d01eacba62ee0cda20a0c23) 

Covers the case, where timestamps are missynchronized:
```
clock: 0 timestamp: 31.12.2022 10:00:00
clock: 1 timestamp: 01.01.2023 23:59:30
clock: 2 timestamp: 01.01.2023 23:59:45
clock: 3 timestamp: 02.01.2023 00:00:05
clock: 4 timestamp: 02.01.2023 00:00:10
clock: 5 timestamp: 01.01.2023 23:59:55
```

Before, it would result in repeated labels:
```
clock: 0 timestamp: 31.12.2022 10:00:00
    LABEL: 1.01.2023
clock: 1 timestamp: 01.01.2023 23:59:30
clock: 2 timestamp: 01.01.2023 23:59:45
    LABEL: 2.01.2023
clock: 3 timestamp: 02.01.2023 00:00:05
clock: 4 timestamp: 02.01.2023 00:00:10
    LABEL: 1.01.2023
clock: 5 timestamp: 01.01.2023 23:59:55
```

fixes: https://github.com/status-im/status-desktop/issues/8962

**Comment by osmaczko** [comment]
Also CCing: @alexjba in case this is merged before https://github.com/status-im/status-desktop/pull/9031. Please be aware of [fix(chat/messages): ensure there is only one label for given date](https://github.com/status-im/status-desktop/pull/9146/commits/41213486eb39acc25d01eacba62ee0cda20a0c23) when (if) resolving conflicts.

**Review by caybro — APPROVED** [review]
Looks good

**Review by saledjenic — APPROVED** [review]

**Comment by osmaczko** [comment]
@jrainville could you please briefly test this PR? QAs involvement is not required here imo.

**Review by jrainville — APPROVED** [review]

**Comment by jrainville** [comment]
Testing now

**Comment by jrainville** [comment]
Tested. It looks good to me. There is no exact repro steps, but I joined two big channels (`#test` and `#testjo`) and all the timestamps were in order. The messages even came in in a weird order because of scroll + mailserver, but the timestamp order was always consistent

**Comment by osmaczko** [comment]
> Tested. It looks good to me. There is no exact repro steps, but I joined two big channels (`#test` and `#testjo`) and all the timestamps were in order. The messages even came in in a weird order because of scroll + mailserver, but the timestamp order was always consistent

Great. Thanks!

**Pull request merged**